### PR TITLE
Change DBus path stratis1 to stratis2 in preparation for new version

### PR DESCRIPTION
**Pull request by jbaublitz**
Closes stratis-storage/project#103

### CODE REVIEW

**Comment by mulkieran** [comment]
@jbaublitz Please rebase when you get the chance.

File: tests/blackbox/stratisd_cert.py [hunk 1]
```diff
@@ -90,8 +90,8 @@ def test_get_managed_objects(self):
             [
                 "busctl",
                 "call",
-                "org.storage.stratis1",
-                "/org/storage/stratis1",
+                "org.storage.stratis2",
```
Comment (mulkieran) [inline tests/blackbox/stratisd_cert.py]: Please use constants defined in testlib/dbus.py here, instead of literal strings and make them public.

Comment (mulkieran) [reply tests/blackbox/stratisd_cert.py]: We realized that we have a fine method for doing this already in testlib/dbus.py. So we'll call that method, instead, and these string literals will just go away as a result.

Comment (jbaublitz) [reply tests/blackbox/stratisd_cert.py]: I've switched `_get_managed_objects()` to `get_managed_objects()` and am using that instead of busctl. Resolved.

**Comment by jbaublitz** [comment]
@mulkieran @bgurney-rh Rebased and ready to go.

**Pull request merged**